project-base/docker/php-fpm/Dockerfile

```diff
@@ -79,6 +79,23 @@ RUN composer global require hirak/prestissimo
 
 ########################################################################################################################
 
+FROM base as development
+
+USER root
+
+# allow overwriting UID and GID o the user "www-data" to help solve issues with permissions in mounted volumes
+# if the GID is already in use, we will assign GID 82 instead (82 is the standard uid/gid for "www-data" in Alpine)
+ARG www_data_uid=82
+ARG www_data_gid=82
+RUN deluser www-data && (addgroup -g $www_data_gid www-data || addgroup -g 82 www-data) && adduser -u $www_data_uid -D -S -G www-data www-data
```

Comment:
In sake of consistency, I would suggest making the arguments optional in the same way we do it in microservices: https://github.com/shopsys/shopsys/blob/65c68a133161f0bf3848db2795359e5412fc263d/microservices/product-search/docker/Dockerfile#L198-L202

This would of course require rewriting the upgrade note to something like "_make the `www_data_*id` arguments optional using an `if` condition (for building `ci` and `production` stage)_".